### DISCUSSION TITLE
How do I do an official release after a prerelease?

### DISCUSSION THREAD

**Discussion by tgoze** [microsoft/beachball#798] [Q&A]
I'm having trouble understanding how you would release a package after it has been published with the `--prerelease-prefix` option.

I would want all of my pre release changes inside my official release change log. For example, if I were to have several pre releases like 1.1.0-alpha.1 all the way to 1.1.0-alpha.5 and I went to do an official release I would want 1.1.0 to have all changes that the 5 alphas had. But the way beachball works is that is removes all changelogs when pre releasing... so when I go to do an official release, the `publish` command doesn't detect any changes and won't bump/publish my 1.1.0 version.

Is there something I'm missing here about expected workflow/design? Clarification/documentation on this would be very helpful.

**Comment by rdebeasi** [comment]
It looks like removing the `prereleasePrefix` option from `beachball.config.js` allows me to "exit pre-release mode." I can then create non-prerelease changes.

That said, if I create a `minor` change, I'll go from 1.1.0-alpha.5 to 1.2.0. As you pointed out, we'd want to instead publish `1.1.0` as stable. If I manually edited `package.json` to roll back the version to 1.0.0 and then created a `minor` change, I think the result would be a stable 1.1.0. That said, editing package numbers manually seems error-prone, so I'd love to hear about an alternative!
